.goreleaser.yaml

```
@@ -0,0 +1,63 @@
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    main: ./cmd/lava
+    goarch:
+      - amd64
+      - arm
+      - arm64
```

I would also add `386`.

```suggestion
    goarch:
      - 386
      - amd64
      - arm
      - arm64
```